opcua/server/user_manager.py

```diff
@@ -7,12 +7,14 @@
 try:
     from opcua.crypto import uacrypto
 except ImportError:
-    logging.getLogger(__name__).warning("cryptography is not installed, use of crypto disabled")
     use_crypto = False
 
 
 class UserManager(object):
 
+    if use_crypto is False:
+        logging.getLogger(__name__).warning("cryptography is not installed, use of crypto disabled")
+
```

Comment:
if someone uses user_manager, they will also use Server so no need to print an extra warning in that file